source/process/spec-process.rst

```diff
@@ -0,0 +1,46 @@
+Spec Review Process
+===================
+
+This is a rough guideline for the process for drafting and reviewing specs. It’s meant to be used as a starting point, but not all specs will follow this exact process - please use your judgment, as it’s something that will vary depending on the situation.
+
+1. Brainstorm / research.
+    - Research topic
+    - Research customer requirements
+    - Pair design brainstorm
+    - User workflows
+    - Consider options / vision for UI
+2. Initial draft.
+    - Goal / “Why"
+
+      - Link to customer data / feedback
+      - Link back to company objectives
+    - User Stories
+    - Rough and Quick Wireframes / Mock ups
+
+      - Flows of screens
+      - Focus should be on the mock ups, with text that supports design
+
+    - Questions
```

Comment:
Updated to explain a bit more, left success metrics because I think that's pretty standard? 

This is for the core team handbook, so the audience would be mainly new hire product managers / ux designers.